test/reporter.js

```diff
@@ -26,7 +26,7 @@ test("✈  reporter", (t) => {
 })
 
 test("✈  timeInfo", (t) => {
-  const timeInfo = require("../src/reporter/timeInfo")
+  const timeInfo = require("../src/reporter/timeInfo").default
```

Comment:
In Babel 6, the `default` is not exported at top level when using `require` for security reason. We have to explicitely add `.default`. [See here](http://webcache.googleusercontent.com/search?q=cache:i9V50YCRtT0J:github.com.proxy.parle.co/babel/babel/issues/2212+&cd=3&hl=fr&ct=clnk&gl=fr) (sorry, this is only available from Google cache). There is [a plugin](https://www.npmjs.com/package/babel-plugin-add-module-exports) to patch that, but obviously [there is also a bug](https://phabricator.babeljs.io/T6644) that make plugins conflicting. :+1: It's not that problematic, as this is only affecting code `require`'ing ES6 exports. ES6 import works fine.